services-gateway-tests/src/main/java/io/scalecube/services/testservice/SecuredWsGwSessionManager.java

```diff
@@ -0,0 +1,29 @@
+package io.scalecube.services.testservice;
+
+import io.scalecube.services.gateway.GatewaySession;
+import io.scalecube.services.gateway.SessionEventHandler;
+import io.scalecube.services.gateway.ws.GatewayMessage;
+
+public class SecuredWsGwSessionManager implements SessionEventHandler<GatewayMessage> {
+  private final AuthRegistry authRegistry;
+
+  public SecuredWsGwSessionManager(AuthRegistry authRegistry) {
+    this.authRegistry = authRegistry;
+  }
+
+  @Override
+  public GatewayMessage mapMessage(GatewaySession session, GatewayMessage req) {
+    return GatewayMessage.from(req).header(AuthRegistry.SESSION_ID, session).build();
+  }
+
+  @Override
+  public void onSessionOpen(GatewaySession s) {
+    System.out.println("Session opened: " + s);
```

Comment:
why println?

Reply:
all these classes are for tests, to demonstrate the usage of auth framework,  and println - just to see what's happening inside while running tests

Reply:
@snripa indeed. IMO it's more convenient to use logger api.